services/api-server/tests/unit/test_api_solver_jobs.py

```diff
@@ -222,15 +237,19 @@ def _put_pricing_plan_and_unit_side_effect(
         assert int(path_params["pricing_unit_id"]) == _pricing_unit_id
         return capture.response_body
 
+    callbacks = [
+        _get_job_side_effect,
+        _put_pricing_plan_and_unit_side_effect,
+        _start_job_side_effect,
+    ]
+    if expected_status_code == 200:
```

Comment:
MINOR: I like to keep all codes with ``fastapi.status`, i.e.
```python
from fastapi import status

if expected_status_code == status.HTTP_200_OK:
   ...
```

Reply:
Done